app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
-
+    people = {}
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    res_lis = [Person(person["name"], person["age"]) for person in people]
```

Comment:
This line creates `Person` objects for each entry in the `people` list. Ensure that the input list contains unique names to prevent overwriting in the `Person.people` dictionary.